x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -1710,4 +1710,37 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'windows.advanced.events.event_on_access.file_paths',
+    first_supported_version: '8.15',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.windows.advanced.events.event_on_access.file_paths',
+      {
+        defaultMessage:
+          'Comma-separated list of additional wildcard patterns that will be monitored for read access. Endpoint will report at most one match per pattern per process. Endpoint will attempt to convert drive letters to NT paths (e.g. \\Device\\HarddiskVolume4), but conversion will fail for per-user drives such as network drives. Put only commas (no spaces) between entries. Wildcard matching is case-insensitive. See Microsoft FsRtlIsNameInExpression documentation for wildcard matching rules.',
+      }
+    ),
+  },
+  {
+    key: 'windows.advanced.events.event_on_access.registry_paths',
+    first_supported_version: '8.15',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.windows.advanced.events.event_on_access.registry_paths',
+      {
+        defaultMessage:
+          'Comma-separated list of registry paths that will be monitored for read access. These must be NT paths (e.g. \\REGISTRY\\MACHINE\\SOFTWARE\\Microsoft\\...). Endpoint will report at most one match per pattern per process. Put only commas (no spaces) between entries. Wildcard matching is case-insensitive. See Microsoft FsRtlIsNameInExpression documentation for wildcard matching rules.',
+      }
+    ),
+  },
+  {
+    key: 'mac.advanced.events.event_on_access.file_paths',
+    first_supported_version: '8.15',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.mac.advanced.events.event_on_access.file_paths',
+      {
+        defaultMessage:
+          'Comma-separated list of additional wildcard patterns that will be monitored for read access. TODO FILL IN THE REST OF DESCRIPTION.',
```

Comment:
@ricardoungureanu When you have time, could you give me a description for this?

Reply:
Thanks.  Is there any sort of deduping like what's described for the Windows versions above?

Reply:
```suggestion
          'Comma-separated list of additional wildcard patterns that will be monitored for read access.  Put only commas (no spaces) between entries. Wildcard matching is case-insensitive.',
```